### PR TITLE
Italian Language

### PR DESCRIPTION
**Pull request by AlexandreVea**
All Italian Language for Kobralost and Zero's Addons.

### CODE REVIEW

File: zeros_farmer/zfa_languages/sh_language_it.lua
Comment (hugoheml) [inline zeros_farmer/zfa_languages/sh_language_it.lua:1]: You need to change the "fr" to "it" here

**Review by hugoheml — CHANGES_REQUESTED** [review]
Hey,
Thanks for your contribution!
In order to release it, you just need to fix a little problem

File: zeros_farmer/zfa_languages/sh_language_it.lua
Comment (AlexandreVea) [reply zeros_farmer/zfa_languages/sh_language_it.lua:1]: It's good !

**Review by hugoheml — APPROVED** [review]
Perfect, thanks for your updates

**Pull request merged**